docs/content/installation/integrations/app-protect-waf-v5/installation.md

```diff
@@ -346,3 +346,15 @@ If you prefer not to build your own NGINX Ingress Controller image, you can use
 
 - Download the image using your NGINX Ingress Controller subscription certificate and key. View the [Get NGINX Ingress Controller from the F5 Registry]({{< relref "installation/nic-images/get-registry-image.md" >}}) topic.
 - The [Get the NGINX Ingress Controller image with JWT]({{< relref "installation/nic-images/get-image-using-jwt.md" >}}) topic describes how to use your subscription JWT token to get the image.
+
+---
+
+## [NGINX App Protect WAF v5 version](https://docs.nginx.com/nginx-app-protect-waf/v5/releases/)
+
+{{< bootstrap-table "table table-bordered table-striped table-responsive" >}}
+| NIC Version | App Protect WAFv5 Version | Config Manager | Enforcer |
+| --- | --- | --- | --- |
+| 3.6.2 | 32_5.48 | 5.2.0 | 5.2.0 |
+{{% /bootstrap-table %}}
+
+{{< note >}} Recompilation of the WAF policies upon every NIC release is recommended in order to get updates to features like threat campaigns and signature updates etc.{{< /note >}}
```

Comment:
```suggestion
{{< note >}} It is recommended to re-compile your AppProtect WAF Policy Bundles with each release of NGINX Ingress Controller. This will ensure your Policies remain compatible, and are compiled with the latest Attack Signatures, Bot Signatures, and Threat Campaigns.{{< /note >}}
```

Reply:
@ADubhlaoich how do you feel about this re-wording suggestion?

Reply:
```suggestion
{{< note >}} F5 recommends to re-compile your NGINX AppProtect WAF Policy Bundles with each release of NGINX Ingress Controller. This will ensure your Policies remain compatible, and are compiled with the latest Attack Signatures, Bot Signatures, and Threat Campaigns.{{< /note >}}
```

Reply:
updated based on @shaun-nx 's suggestion